openqabot/types/aggregate.py

```diff
@@ -60,7 +61,9 @@ def __call__(
         ignore_onetime: bool = False,
     ) -> List[Dict[str, Any]]:
         ret = []
-
+        log.debug(
+            "Generating aggregate for %s the following incidents: %s", self, incidents
```

Comment:
I don't understand that sentence. The object reference `self` would be put into the first `%s`? How would an example log line look like?

Reply:
https://github.com/openSUSE/qem-bot/actions/runs/7724416299/job/21056528243?pr=154#step:5:185

Reply:
Got it, so

```
bot.types.aggregate:aggregate.py:64 Generating aggregate for <Aggregate product: > the following incidents: [<MockIncident id: 1284 with repos [Repos(product='BBBBBBBBB', version='CCCCCCCC', arch='ciao')]>]
```

not a perfect format but for a "debug" message acceptable